packages/settings/src/screens/access/index.tsx

```
@@ -25,24 +25,18 @@ const Access = () => {
 		<>
 			<div className="wcpos:p-4">
 				<Notice status="info" isDismissible={false}>
-					<UT
-						_str="By default, access to the POS is limited to Administrator, Shop Manager and Cashier roles.
-					It is recommended that you <strong>do not change</strong> the default settings unless you
-					are fully aware of the consequences."
-						_tags="wp-admin-settings"
-						_inline
+					<Trans
+						i18nKey="By default, access to the POS is limited to Administrator, Shop Manager and Cashier roles. It is recommended that you <1>do not change</1> the default settings unless you are fully aware of the consequences."
+						components={[, <strong />]}
```

Biome (2.3.13)

[error] This array contains an empty slots..

The presences of empty slots may cause incorrect information and might be a typo.
Unsafe fix: Replace hole with undefined


(lint/suspicious/noSparseArray)